templates/documents/copy.html.twig

```diff
@@ -0,0 +1,27 @@
+{% extends 'layout-with-submenu.html.twig' %}
+
+{% block title %}{% trans %}Copy document to other cases{% endtrans %}{% endblock %}
+
+{% block header %}
+    <div class="row">
+        <div class="col">
+            <h1>{% trans %}Basic information{% endtrans %}</h1>
+        </div>
+        <div class="col-auto">
+            <a name="" id="" class="btn btn-secondary" href="{{ url('case_documents', {id: case.id}) }}" role="button">{% trans %}Cancel{% endtrans %}</a>
+        </div>
+    </div>
+
+    {{ form_start(copy_document_form) }}
+    <form class="form" >
+        <div class="form-row">
+            <div class="col-md-2 mb-3">
+                <label >{% trans %}Cases{% endtrans %}</label>
+                {{ form_widget(copy_document_form.cases, {'attr': {'class': 'form-control', 'required': true}}) }}
+            </div>
+        </div>
+        {{ form_widget(copy_document_form.save, {'attr': {'class': 'btn btn-primary', 'type': 'submit'}}) }}
+        {{ form_rest(copy_document_form) }}
+    </form>
+    {{ form_end(copy_document_form) }}
```

Comment:
I think this will produce two `</form>` tags?

Reply:
Yes. Avoided duplicate `<form>` elements.